README.md

```diff
@@ -3,7 +3,7 @@
 [![Tests](https://github.com/qgis/QGIS/workflows/QGIS%20tests/badge.svg)](https://github.com/qgis/QGIS/actions/workflows/run-tests.yml)
 [![Docker Status](https://img.shields.io/docker/automated/qgis/qgis.svg)](https://hub.docker.com/r/qgis/qgis/tags)
 [![Build Status](https://dev.azure.com/qgis/QGIS/_apis/build/status/qgis.QGIS?branchName=master)](https://dev.azure.com/qgis/QGIS/_build/latest?definitionId=1&branchName=master)
-![Windows cross build][https://github.com/qgis/QGIS/workflows/MingW64%20Windows%2064bit%20Build/badge.svg](https://github.com/qgis/QGIS/actions/workflows/mingw64.yml)
+[![Windows cross build](https://github.com/qgis/QGIS/workflows/MingW64%20Windows%2064bit%20Build/badge.svg)](https://github.com/qgis/QGIS/actions/workflows/mingw64.yml)
```

Comment:
also wonder: should this not be filtered to the target branch? IOW would this return the result of the last test of this action (including PR) or the last run within the master branch?
```suggestion
[![Windows cross build](https://github.com/qgis/QGIS/workflows/MingW64%20Windows%2064bit%20Build/badge.svg)](https://github.com/qgis/QGIS/actions/workflows/mingw64.yml?query=branch%3Amaster+event%3Apush)
```

Reply:
For reference, it seems not to be possible to adjust this dynamically
https://github.com/github/markup/issues/913

Reply:
Yep! "target branch" is inaccurate as, unless manually fixed, they will all point master status.
My point here is more related to "broken" PR, or another broken branch (shouldn't happen...) that is last run. The failing result would then display on master readme. no?